server/models/userModels.js

```diff
@@ -4,88 +4,93 @@ import validator from 'validator';
 import bcrypt from 'bcrypt';
 
 const userSchema = new mongoose.Schema({
-	email: {
-		type: String,
-		required: [true, 'An email is required.'],
-		validate: {
-			validator: function (val) {
-				return validator.isEmail(val);
-			},
-			message: 'Please enter a valid email.',
-		},
-		unique: true,
-	},
-	password: {
-		type: String,
-		required: [true, 'A password is required'],
-		validate: {
-			validator: function (val) {
-				return validator.isStrongPassword(val, {
-					minLength: 6,
-					minLowercase: 1,
-					minUppercase: 0,
-					minNumbers: 1,
-					minSymbols: 0,
-				});
-			},
-			message:
-				'A password must be at least 6 charactors including at least one letter and number.',
-		},
-	},
-	date: {
-		type: Date,
-		default: new Date(),
-	},
-	name: {
-		type: String,
-	},
-	publicEmail: {
-		type: String,
-		validate: {
-			validator: function (val) {
-				return validator.isEmail(val);
-			},
-			message: 'Please enter a valid email.',
-		},
-	},
-	senpaiProfile: {
-		id: {
-			type: String,
-			default: uuidv4(),
-		},
-		isActive: {
-			type: Boolean,
-		},
-		description: {
-			type: String,
-		},
-		techStack: {
-			type: Array,
-		},
-	},
-	kohaiProfile: {
-		id: {
-			type: String,
-			default: uuidv4(),
-		},
-		isActive: {
-			type: Boolean,
-		},
-		description: {
-			type: String,
-		},
-		techStack: {
-			type: Array,
-		},
-	},
+  email: {
+    type: String,
+    required: [true, 'An email is required.'],
+    validate: {
+      validator: function (val) {
+        return validator.isEmail(val);
+      },
+      message: 'Please enter a valid email.',
+    },
+    unique: true,
+  },
+  password: {
+    type: String,
+    required: [true, 'A password is required'],
+    validate: {
+      validator: function (val) {
+        return validator.isStrongPassword(val, {
+          minLength: 6,
+          minLowercase: 1,
+          minUppercase: 0,
+          minNumbers: 1,
+          minSymbols: 0,
+        });
+      },
+      message:
+        'A password must be at least 6 charactors including at least one letter and number.',
+    },
+  },
+  date: {
+    type: Date,
+    default: new Date(),
+  },
+  name: {
+    type: String,
+    default: '',
+  },
+  publicEmail: {
+    type: String,
+    validate: {
+      validator: function (val) {
+        return validator.isEmail(val);
+      },
+      message: 'Please enter a valid email.',
+    },
+  },
+  senpaiProfile: {
+    id: {
+      type: String,
+      default: uuidv4(),
+    },
+    isActive: {
+      type: Boolean,
+      default: false,
+    },
+    description: {
+      type: String,
+      default: '',
```

Comment:
Added a default value for description so we can at least have the key and empty string on the setting page